@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardHostsPatches.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <UiCard>
+    <UiCardTitle class="patches-title">
+      {{ $t("patches") }}
+      <template v-if="areAllLoaded" #right>
+        {{ $t("missing", { n: count }) }}
+      </template>
+    </UiCardTitle>
+    <div class="table-container">
+      <HostPatches :hosts-loaded-status="loadedStatus" :patches="patches" />
+    </div>
+  </UiCard>
+</template>
+
+<script lang="ts" setup>
+import HostPatches from "@/components/HostPatchesTable.vue";
+import UiCard from "@/components/ui/UiCard.vue";
+import UiCardTitle from "@/components/ui/UiCardTitle.vue";
+import { useHostPatches } from "@/composables/host-patches.composable";
+import { useHostStore } from "@/stores/host.store";
+import { computed } from "vue";
+
+const { records: hosts } = useHostStore().subscribe();
+
+const { loadedStatus, count, patches } = useHostPatches(() =>
+  hosts.value.map((host) => host.$ref)
+);
+
+const areAllLoaded = computed(() => {
+  return [...loadedStatus.value.values()].every((isReady) => isReady);
+});
+</script>
+
+<style lang="postcss" scoped>
+.patches-title {
+  --card-title-right-color: var(--color-red-vates-base);
+}
+
+.table-container {
+  max-height: 40rem;
+  overflow: scroll;
```

Comment:
![Capture d’écran de 2023-03-21 16-01-03](https://user-images.githubusercontent.com/70369997/226648038-43000b14-49a6-4b6e-b3ab-44ddad3fc160.png)

`overflow: scroll` also adds a scrollbar on the `x` axis even if there is no scroll.

```suggestion
  overflow-y: scroll;
```

IMO, if there's no need to scroll (not enough data), we shouldn't show a scrollbar.

```suggestion
  overflow: auto;
```